static/js/components/LocalizationPlot.jsx

```diff
@@ -136,11 +139,259 @@ LocalizationPlot.propTypes = {
       processed_fraction: PropTypes.number,
     })
   ).isRequired,
-  options: PropTypes.arrayOf(PropTypes.number),
+  options: PropTypes.arrayOf(PropTypes.number).isRequired,
 };
 
-LocalizationPlot.defaultProps = {
-  options: [false, false, false, false, false],
+const useD3 = (
```

Comment:
This feels like an abuse of `useD3`.  How about signature:

```
useD3 = (renderer, height, width, data)
```

instead, where `data` can contain anything you want. The idea is that `useD3` could be used more widely than just this module, but that then the renderer and the input data would change.

Reply:
Yes this is a nice idea.